chapters/ch08.md

```diff
@@ -2,14 +2,20 @@
 
 ->
 
+# Thiết lập một phép đo để đánh giá bằng một con số để nhóm của bạn tối ưu hoá 
+
 Classification accuracy is an example of a ​**single-number evaluation metric**​: You run your classifier on the dev set (or test set), and get back a single number about what fraction of examples it classified correctly. According to this metric, if classifier A obtains 97% accuracy, and classifier B obtains 90% accuracy, then we judge classifier A to be superior.
 
 -> 
 
+Độ chính xác trong phân loại là một ví dụ của **một phép đo đánh giá bằng một con số**: bạn chạy thuật toán phân loại của bạn trên một tập phát triển (hoặc tập kiểm tra), rồi nhận được một con số thể hiện số phần trăm mẫu thử được phân loại chính xác. Dựa vào phép đo này, nếu thuật toán phân loại A thu được 97% độ chính xác, và thuật toán phân loại B thu được 90% độ chính xác, thì chúng ta đánh giá cao thuật toán A hơn.
```

Comment:
> ~~bạn~~ Khi chạy thuật toán phân loại ~~của bạn~~ trên một tập phát triển (hoặc tập kiểm tra), ~~rồi~~ và nhận ~~được~~ về **một con số thể hiện số phần trăm mẫu thử được phân loại chính xác.**  
câu in đậm dịch có vẻ chưa chuẩn lắm. 

> Dựa vào phép đo này, nếu thuật toán phân loại A thu được 97% **độ chính xác**~~,~~ và thuật toán phân loại B thu được 90% độ chính xác, thì chúng ta đánh giá cao thuật toán A hơn.
- Cần thống nhất lại thuật ngữ accuracy bên glossary. 
- không có dấu phẩy trước chữ và 
- nên sửa _thì chúng ta đánh giá cao thuật toán A hơn._ cho mượt hơn xí

Reply:
Em đã sửa ở bản commit gần nhất ạ